sdk/src/client/api/block_builder/input_selection/requirement/mana.rs

```diff
@@ -259,6 +260,22 @@ impl InputSelection {
         Ok(added_inputs)
     }
 
+    pub(crate) fn initial_mana_excess(&self) -> Result<u64, Error> {
```

Comment:
This doesn't need to consider mana rewards?

Reply:
Ah that's added in total_mana()

Reply:
It does! Check total_mana